docs/api-overview.md

```diff
@@ -24,9 +24,30 @@
 
 # API documentation
 
-The Eclipse OpenJ9 VM provides supplementary application programming interfaces and extensions, which can be used to improve performance, assist with problem determination, or help monitor and manage the OpenJ9 VM.
+The Eclipse OpenJ9 VM provides supplementary application programming interfaces and extensions, which can be used to improve performance, assist with problem determination, or help monitor and manage the OpenJ9 VM. The documentation also includes links to the API documentation for the Java&trade; SE and JDK reference implementation.
+
+
+## Native data operations
+
+If your Java application manipulates native data, the Data Access Accelerator API package (`com.ibm.dataaccess`) can help improve application performance. Classes are available for manipulating packed decimal data, converting between decimal data types, and marshalling Java binary types to and from byte arrays.

```

Comment:
Only change I would suggest is instead of saying "manipulating packed decimal data" I would be more explicit and say something along the lines of:

> performing arithmetic, comparison, and validation of packed decimal data

and add that conversion to and from `BigDecimal` and `BigInteger` is supported:

> converting between decimal data types as well as to/from `BigDecimal` and `BigInteger` types

Reply:
Added - thanks.